scripts/r2s.py

```diff
@@ -183,7 +203,10 @@ def step2():
         tags = {('TOTAL', dc): tag_name}
         photon_source_hdf5_to_mesh(mesh, h5_file, tags, sub_voxel=sub_voxel,
                                    cell_mats=cell_mats)
-        mesh.write_hdf5('{0}_{1}.h5m'.format(output, i+1))
+        p_src_filename = '{0}_{1}.h5m'.format(output, i+1)
+        mesh.write_hdf5(p_src_filename)
+        if transport_code.lower() == 'openmc':
```

Comment:
Does it break anything in MCNP if the filetype attribute is also set there?  Seems like a useful thing to do in all cases.

Reply:
It will not break anything in MCNP. This parameter is currently not used in DAG-MCNP. We can keep it there is possible usage of it.